core/model-api/src/main/java/org/eclipse/rdf4j/model/base/AbstractBNode.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ ******************************************************************************/
+
+package org.eclipse.rdf4j.model.base;
+
+import java.util.Objects;
+
+import org.eclipse.rdf4j.model.BNode;
+
+public abstract class AbstractBNode implements BNode {
```

Comment:
This class, and the other new abstract classes, could use a minimal bit of Javadoc. All it really needs:

> Base class for {@link BNode}, offering common functionality.

and preferable an author tag, and a `@since 3.5.0` tag as well.